core-bundle/tests/EventListener/DataContainer/DefaultOperationsListenerTest.php

```diff
@@ -409,6 +409,38 @@ public function testDoesNotAddDefaultsIfOneOperationHasADefaultKey(): void
         $this->assertSame(['foo'], array_keys($operations));
     }
 
+    public function testSetsPriorityFromStringKey(): void
+    {
+        /** @phpstan-var array $GLOBALS (signals PHPStan that the array shape may change) */
+        $GLOBALS['TL_DCA']['tl_foo'] = [
+            'list' => [
+                'sorting' => [
+                    'mode' => DataContainer::MODE_SORTED,
+                ],
+                'operations' => [
+                    '-edit',
+                    '+show',
```

Comment:
Is this a new feature? How does it work?

Reply:
Yes it is. You somehow need a way to mark a string-only operation as primary. I changed it in 23a7b86c0f45a66d3b50302a4ae3588bb65e8a48 though (reverted to the first draft actually). You now would use `!show` to mark an operation as primary. String operations are never _primary_ by default.